lomap/dbmol.py

```diff
@@ -85,6 +86,28 @@ def ecr(mol_i, mol_j):
     return scr_ecr
 
 
+def _find_common_core(mols, element_change: bool) -> str:
+    """Find common core among input molecules to speed up future MCS"""
+    # strip hydrogens off
+    mols2 = [Chem.RemoveHs(m) for m in mols]
+
+    if element_change:
+        atom_compare = rdFMCS.AtomCompare.CompareAny
+    else:
+        atom_compare = rdFMCS.AtomCompare.CompareElements
+
+    res = rdFMCS.FindMCS(mols2,
+                       timeout=60,
```

Comment:
Could we ever have a case where this timeout is reached? I.e. would it be worth propagating the timeout from the user input?

Reply:
fixed this